src/pvsite_datamodel/read/forecast_value.py

```diff
@@ -117,6 +117,59 @@ def get_forecast_values_fast(
     return forecast_values
 
 
+def get_forecast_values_day_ahead_fast(
```

Comment:
I wasnt sure if I should put this a seperate function or combine in, but I think i prefer it seperate. Makes it a bit more readable